applications/SwimmingDEMApplication/swimming_DEM_application.h

```diff
@@ -41,6 +42,9 @@
 #include "../DEMApplication/custom_elements/spheric_particle.h"
 #include "../DEMApplication/custom_elements/nanoparticle.h"
 #include "../DEMApplication/custom_elements/analytic_spheric_particle.h"
+#include "../FluidDynamicsApplication/custom_elements/qs_vms.h"
+#include "../FluidDynamicsApplication/custom_utilities/qsvms_data.h"
+#include "custom_utilities/qsvmsdemcoupled_data.h"
```

Comment:
```suggestion
#include "custom_utilities/qsvms_dem_coupled_data.h"
```
I'd rename it to follow the standard convention.